ci/build.ps1

```diff
@@ -0,0 +1,125 @@
+Set-PSDebug -Trace 1
+
+$env:CONFIGURATION -match '^(?<compiler>\w+)_(?<bits>32|64)(?:-(?<option>\w+))?$'
+$compiler = $Matches.compiler
```

Comment:
Should these be script-scoped? E.g. `$script:compiler`. I usually do this to be explicit, the implicit rules can surprise people.